python/modules/IcePy/Proxy.cpp

```diff
@@ -2062,18 +1888,10 @@ static PyMethodDef ProxyMethods[] = {
      reinterpret_cast<PyCFunction>(proxyIceInvokeAsync),
      METH_VARARGS | METH_KEYWORDS,
      PyDoc_STR("ice_invokeAsync(op, mode, inParams[, context]) -> Ice.Future")},
-    {"ice_checkedCast",
-     reinterpret_cast<PyCFunction>(proxyIceCheckedCast),
-     METH_VARARGS | METH_CLASS,
-     PyDoc_STR("ice_checkedCast(proxy, id[, facetOrContext[, context]]) -> proxy")},
```

Comment:
Why do we implement uncheckedCast in C++? (If fix needed, can wait until a follow-up PR).

Reply:
Currently we cannot create a new proxy from another proxy in Python side. ObjectPrx::createProxy is not exposed to Python.